package.json

```diff
@@ -4,6 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "start": "NODE_ENV=dev nodemon ./src/index.js --exec babel-node --",
```

Comment:
@ivymwende i think `NODE_ENV=production` is clear

Reply:
@NiyoEric What do you mean by this?